tests/test_padding_oracle.py

```diff
@@ -14,6 +15,21 @@ def test_padding_oracle_basic():
 
     assert decrypted == plaintext
 
+def test_padding_oracle_encryption():
+    cryptor = VulnerableCryptor()
+
+    plaintext = b'the quick brown fox jumps over the lazy dog'
+    ciphertext = cryptor.encrypt(plaintext)
+
+    padder = padding.PKCS7(128).padder()
```

Comment:
This script already supports only PKCS7 padding - maybe its a good idea shift the responsibility of padding the payload from user to the library?